src/ws/session.c

```diff
@@ -366,9 +366,11 @@ static bool
 acquire_service_credentials (gss_OID mech_type, gss_cred_usage_t usage, gss_cred_id_t *cred)
 {
   /* custom credential store with our cockpit keytab */
-  static gss_key_value_element_desc store_elements[] = { { .key = "keytab", .value = COCKPIT_KTAB, },
-                                                         { .key = "ccache", .value = "MEMORY:", } };
-  static const gss_key_value_set_desc cockpit_ktab_store = { .count = 2, .elements = store_elements };
+  gss_key_value_element_desc store_elements[] = {
+      { .key = (usage == GSS_C_INITIATE) ? "client_keytab" : "keytab", .value = COCKPIT_KTAB, },
```

Comment:
I'm not fully happy with this. Either let's turn `usage` into a `bool is_client`, or explicitly pass the keytab type as an argument.

Reply:
On second thought, a boolean is hard to read in calls, and it's not actually bad to centralize the knowledge of how to map usages to config keys here. Let's see what other reviewers say.